libs/designsystem/src/lib/components/avatar/avatar.component.ts

```diff
@@ -29,4 +30,13 @@ export class AvatarComponent {
   get _cssClass() {
     return [this.themeColor, this.size].filter((cssClass) => !!cssClass);
   }
+
+  ngOnInit(): void {
+    if (this.shadow) {
```

Comment:
Consider a "return early" approach here: https://medium.com/swlh/return-early-pattern-3d18a41bba8
This makes the code easier to read IMO.
So it will look like this: 

```javascript
public ngOnInit(): void {
    if (!this.shadow) {
     return;
    }
    
    this.stroke = true;
    console.warn(
      'Shadow input binding on avatar will be deprecated next major. Use stroke instead'
      );
}
```

Reply:
It does look better! Thank you!